wled00/FX_fcn.cpp

```diff
@@ -1432,7 +1434,11 @@ bool WS2812FX::isUpdating() const {
  */
 uint16_t WS2812FX::getFps() const {
   if (millis() - _lastShow > 2000) return 0;
-  return _cumulativeFps +1;
+  #ifdef WLED_DEBUG
+  return (FPS_MULTIPLIER * (_cumulativeFps + (random16() & 63))) >> 7;  // + random("0.5") for dithering
+  #else
+  return (FPS_MULTIPLIER * _cumulativeFps) >> 7; // _cumulativeFps is stored in fixed point 9.7 bit
```

Comment:
Maybe replace all places with "7" by a constant (#define) ?

 Its better for maintainability to leave a hint what the magic number "7" stands for - in this case the fixed point multiplier.

Reply:
done.